op-node/flags/flags.go

```diff
@@ -63,6 +63,12 @@ var (
 		Usage:  "Trust the L1 RPC, sync faster at risk of malicious/buggy RPC providing bad or inconsistent L1 data",
 		EnvVar: prefixEnvVar("L1_TRUST_RPC"),
 	}
+	L1RPCProviderKind = cli.StringFlag{
+		Name:   "l1.rpckind",
+		Usage:  "The kind of RPC provider, used to inform optimal transactions receipts fetching, and thus reduce costs. Valid options: alchemy, quicknode, infura, parity, nethermind, debug_geth, erigon, basic, any.",
```

Comment:
`RPCKinds.String()` would be helpful here to prevent this from getting out of date